From b55266d42cbf89e150cee1d4dec4bb5f2fd78c49 Mon Sep 17 00:00:00 2001
From: William Douglas <william.douglas@intel.com>
Date: Tue, 17 Jan 2017 17:51:09 +0000
Subject: [PATCH] Add make install for pigz

---
 Makefile | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/Makefile b/Makefile
index 5d8ea89..3b45b4a 100644
--- a/Makefile
+++ b/Makefile
@@ -81,5 +81,12 @@ docs: pigz.pdf
 pigz.pdf: pigz.1
 	groff -mandoc -f H -T ps pigz.1 | ps2pdf - pigz.pdf
 
+install: pigz
+	mkdir -p $(DESTDIR)/usr/bin
+	cp pigz $(DESTDIR)/usr/bin/pigz
+	ln -s pigz $(DESTDIR)/usr/bin/gzip
+	mkdir -p $(DESTDIR)/usr/share/man/man1/
+	cp pigz.1 $(DESTDIR)/usr/share/man/man1/pigz.1
+
 clean:
 	@rm -f *.o ${ZOPFLI}*.o pigz unpigz pigzn pigzt pigz.c.gz pigz.c.zz pigz.c.zip
-- 
2.11.0

