From 8d4f58ffc8faa3636f775ef1b1f66c5331f9ee9d Mon Sep 17 00:00:00 2001
From: Francisco Boni <boboniboni@gmail.com>
Date: Thu, 12 Aug 2021 06:39:16 -0300
Subject: [PATCH 1/6] rsync

---
 pigz.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/pigz.c b/pigz.c
index 4c1bd1b..75a318d 100644
--- a/pigz.c
+++ b/pigz.c
@@ -4324,10 +4324,10 @@ local void defaults(void) {
     g.procs = nprocs(8);
     g.shift = x2nmodp(g.block, 3);
 #endif
-    g.rsync = 0;                    // don't do rsync blocking
+    g.rsync = 1;                    // don't do rsync blocking
     g.setdict = 1;                  // initialize dictionary each thread
     g.verbosity = 1;                // normal message level
-    g.headis = 3;                   // store name and time (low bits == 11),
+    g.headis = 1;                   // store name and time (low bits == 11),
                                     // restore neither (next bits == 00),
                                     // where 01 is name and 10 is time
     g.pipeout = 0;                  // don't force output to stdout
-- 
2.34.0

