--- pigz-2.3.4/Makefile~	2016-05-30 05:09:41.000000000 +0000
+++ pigz-2.3.4/Makefile	2016-10-03 16:33:47.863641916 +0000
@@ -1,12 +1,12 @@
 CC=cc
-CFLAGS=-O3 -Wall -Wextra
+CFLAGS+=-O3 -Wall -Wextra
 LDFLAGS=
 LIBS=-lm -lpthread -lz
 ZOPFLI=zopfli/src/zopfli/
 # use gcc and gmake on Solaris
 
 pigz: pigz.o yarn.o try.o ${ZOPFLI}deflate.o ${ZOPFLI}blocksplitter.o ${ZOPFLI}tree.o ${ZOPFLI}lz77.o ${ZOPFLI}cache.o ${ZOPFLI}hash.o ${ZOPFLI}util.o ${ZOPFLI}squeeze.o ${ZOPFLI}katajainen.o
-	$(CC) $(LDFLAGS) -o pigz $^ $(LIBS)
+	$(CC) $(CFLAGS) $(LDFLAGS) -o pigz $^ $(LIBS)
 	ln -f pigz unpigz
 
 pigz.o: pigz.c yarn.h try.h ${ZOPFLI}deflate.h ${ZOPFLI}util.h
