--- pigz-2.4/Makefile~	2017-12-27 00:35:01.000000000 +0000
+++ pigz-2.4/Makefile	2017-12-27 16:34:52.785708609 +0000
@@ -1,5 +1,5 @@
 CC=gcc
-CFLAGS=-O3 -Wall -Wextra -Wno-unknown-pragmas
+CFLAGS+=-O3 -Wall -Wextra -Wno-unknown-pragmas
 LDFLAGS=
 LIBS=-lm -lpthread -lz
 ZOPFLI=zopfli/src/zopfli/
@@ -8,7 +8,7 @@
 # use gcc and gmake on Solaris
 
 pigz: pigz.o yarn.o try.o $(ZOP)
-	$(CC) $(LDFLAGS) -o pigz pigz.o yarn.o try.o $(ZOP) $(LIBS)
+	$(CC) $(LDFLAGS) $(CFLAGS) -o pigz pigz.o yarn.o try.o $(ZOP) $(LIBS)
 	ln -f pigz unpigz
 
 pigz.o: pigz.c yarn.h try.h $(ZOPFLI)deflate.h $(ZOPFLI)util.h
